istioctl/cmd/workload.go

```diff
@@ -576,7 +576,7 @@ func createHosts(kubeClient kube.CLIClient, ingressIP, dir string, revision stri
 	}
 
 	var hosts string
-	if net.ParseIP(ingressIP) != nil {
+	if ipa, _ := netip.ParseAddr(ingressIP); ipa.IsValid() {
```

Comment:
we seem to inconsistently check the err vs check IsValid... should we be following the same pattern everywhere?

Reply:
They are equivalent. I think `IsValid` is better then the `err == nil`. unless we need the err， we should use semantically clearer `IsValid`